PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetsEEC.cxx

```diff
@@ -394,14 +394,70 @@ void AliAnalysisTaskJetsEEC::ComputeEEC(AliEmcalJet *fJet, AliJetContainer *fJet
 //Looping over the jet
     double jet_pt = fJet->Pt();
     jet_pt_hist->Fill(jet_pt); //filling histogram with momentum of jets
-//    constit = fConstitutents.constituents(); //vector of single jet constituents. Using pseudojet jet.constituent
+//For jets with 2 constituents
     for(int j=0; j<int(fConstituents.size()); j++)  //looping over constituents of the fConstituents object
         {
+         if(int(fConstituents.size()) == 2)
```

Comment:
To make the logic a bit simpler and more efficient, you can swap the if statement with the outer for loop (and then you can move the next block where you deal with more than 2 constituents into the else block)